virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc

```diff
@@ -1,38 +1,21 @@
 :_content-type: ASSEMBLY
 [id="virt-using-the-default-pod-network-with-virt"]
-= Using the default pod network for virtual machines
+= Configuring the virtual machine for the default pod network
 include::_attributes/common-attributes.adoc[]
 :context: virt-using-the-default-pod-network-with-virt
 
 toc::[]
 
-You can use the default pod network with {VirtProductName}. To do so, you must use the `masquerade` binding method. Do not use `masquerade` mode with non-default networks.
-
-xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#attaching-to-multiple-networks[For secondary networks], use the `bridge` binding method.
+You can connect a virtual machine to the default internal pod network by configuring its network interface to use the `masquerade` binding mode
 
 include::modules/virt-configuring-masquerade-mode-cli.adoc[leveloffset=+1]
 
 include::modules/virt-configuring-masquerade-mode-dual-stack.adoc[leveloffset=+1]
 
-[id="virt-selecting-binding-method"]
-== Selecting binding method
-
-If you create a virtual machine from the {VirtProductName}
-xref:../../../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[web console wizard],
-select the required binding method from the *Networking* screen.
-
-include::modules/virt-networking-wizard-fields-web.adoc[leveloffset=+2]
-
 == Virtual machine configuration examples for the default network
```

Comment:
@bgaydosrh Based on feedback I received on the 2 docs proposal, we can remove this header and the two examples. Then it should be good to go. Thanks!